File: 2-1/프밍실습/기말고사/2번 문항.c
#include <stdio.h>

int main()
{
    int arr[5][5] = {0};
    int k=1;

    for(int i=0; i<5; i++)
        for(int j=4; j>=i; j--)
            arr[i][j] = k++;

    for(int i=0; i<5; i++)
    {
        for(int j=0; j<5; j++)
            printf("%2d ", arr[i][j]);
        printf("\n");
    }
    return 0;
}

File: 2-1/자바/중간고사/2번 문항.java
public class 중간고사_박범수2 {

	public static void main(String[] args) {
		int[][] arr = { {2500000, 84}, {3700000, 73}, {4400000, 54}, {3500000, 92} };
		
    for(int i = 0; i < arr.length; i++)
		{
			if(arr[i][1] >= 80)
				arr[i][0] *= 1.2;
			else if(arr[i][1] >= 60)
				arr[i][0] *= 1.1;
			else
				arr[i][0] *= 0.9;	
		}
	
		for(int money = 0; money < arr.length; money++)
		{
			for(int score = 0; score < arr[money].length; score++)
				System.out.printf("%d ", arr[money][score]);
			System.out.println();
		}
	}
}
